### ISSUE TITLE
The use of Lisflood model parameters (class property) does not match pymt usage

### ISSUE THREAD

**Issue by SarahAlidoost** [eWaterCycle/ewatercycle#60]
According to [pymt documentation](https://pymt.readthedocs.io/en/latest/usage.html), `model.parameters` should return model (hydrological) parameters. This is not the case in the current implementation in the ewatercycle.lisflood.

**Comment by sverhoeven** [comment]
From https://github.com/eWaterCycle/ewatercycle/blob/master/examples/lisflood.ipynb it now returns

```
[('IrrigationEfficiency', '0.8'),
 ('PathRoot',
  '/lustre1/0/wtrcycle/comparison/lisflood_input/Lisflood01degree_masked'),
 ('MaskMap',
  '/lustre1/0/wtrcycle/comparison/recipes_auxiliary_datasets/LISFLOOD'),
 ('config_template',
  '/lustre1/0/wtrcycle/comparison/lisflood_input/settings_templates/settings_lisflood.xml'),
 ('start_time', '1990-01-01T00:00:00Z'),
 ('end_time', '1990-10-10T00:00:00Z'),
 ('forcing directory', '/scratch/shared/ewatercycle/lisflood_forcing_data')]
```

it should return

```
[('IrrigationEfficiency', '0.8'),
 ('MaskMap',
  '/lustre1/0/wtrcycle/comparison/recipes_auxiliary_datasets/LISFLOOD'),
 ('start_time', '1990-01-01T00:00:00Z'),
 ('end_time', '1990-10-10T00:00:00Z'),
]
```
MaskMap should be included if #121 after completed.

**Comment by sverhoeven** [comment]
IrrigationEfficiency, start, end time are hydrological) parameters so that part of issue is already fixed in #84
Will remove paths as part of #113
